packages/serverless-orchestration/src/ServerlessHub.js

```diff
@@ -258,10 +258,13 @@ hub.post("/", async (req, res) => {
 
     await waitForLogger(logger);
     await delay(waitForLoggerDelay); // Wait a few seconds to be sure the the winston logs are processed upstream.
-    res.status(500).send({
-      message: errorOutput instanceof Error ? "A fatal error occurred in the hub" : "Some spoke calls returned errors",
-      output: errorOutput instanceof Error ? errorOutput.message : errorOutput,
-    });
+    res
+      .status(500)
+      .send({
+        message:
```

Comment:
Was this just a formatting change? I can't spot the difference :)

Reply:
ye, and this made the linter fail. I've fixed it and am re-running through CI now.